numpy/core/src/multiarray/textreading/field_types.h

```diff
@@ -31,7 +31,7 @@
  *       considered boolean True or False).
  *
  * TODO: Aside from nailing down the above notes, it may be nice to expose
- *       these function publically.  This could allow user DTypes to provide
+ *       these function publicly.  This could allow user DTypes to provide
```

Comment:
Apparently both are correct but publicly is more commonly used (by 100 to 1 or so), so let's go with that.